src/app/[countryCode]/(main)/layout.tsx

```diff
@@ -14,13 +17,31 @@ export const metadata: Metadata = {
 export default async function PageLayout(props: { children: React.ReactNode }) {
   const customer = await retrieveCustomer()
   const cart = await retrieveCart()
+  let shippingOptions: StoreCartShippingOption[] = []
+
+  if (cart) {
+    const { shipping_options } = await sdk.store.fulfillment.listCartOptions({
```

Comment:
This should be cached, or the endpoint will be called on every pageview.

Create a `listCartOptions` function in `@lib/data/cart`, use `sdk.client.fetch` and pass the proper cache options and a `cache: "force-cache"` header